warehouse/warehouse.go

```diff
@@ -917,7 +918,7 @@ func healthHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func getConnectionString() string {
-	if warehouseMode == config.EmbeddedMode {
+	if warehouseMode == config.EmbeddedMode || warehouseMode == config.PooledWHSlaveMode {
```

Comment:
Let's change the check here to see if warehouse db env variables are present or not, similar to what you are doing with pgnotifier.

Reply:
Done..